sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html

```diff
@@ -275,6 +275,21 @@
     my_FragColor = vec4(1.0 - f, f, 0.0, 1.0);
 }
 </script>
+<script id="fshaderIndexUniform" type="x-shader/x-fragment">#version 300 es
```

Comment:
I don't think it's worth it to exhaustively test all possible variable types, I'd rather just have this targeted regression test.